mlir/test/Dialect/Linalg/transpose-matmul-a.mlir

```diff
@@ -0,0 +1,13 @@
+module attributes {transform.with_named_sequence} {
```

Comment:
This needs `// RUN: mlir-opt %s` to make lit happy.

Reply:
cheers, fixed